spec/features/user_following_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "spec_helper"
+
+feature 'User following' do
+  given(:john) { Fabricate(:user, email: "john@example.com", password: "Password", name: "John Doe") }
+  given(:bob) { Fabricate(:user, email: "bob@codefish.org", password: "Test123", name: "Bob Hope") }
+  given!(:video1) { Fabricate(:video) }
+  given!(:review1) { Fabricate(:review, video: video1, author: bob) }
+
+  background { log_in_user john }
+
+  scenario "start following bob, visit people page and unfollow bob" do
```

Comment:
Rather than lay out the steps in your description, describe it from a high level "user can follow and unfollow bob". If you test unfollowing here though, you don't need the next spec. If you keep the next spec, you don't need to test it here.